packages/components/WhyItWorksSection.tsx

```diff
@@ -13,15 +13,15 @@ import {
 
 import { PersonalizedLearningIcon, PracticalLearningIcon, TeamworkIcon } from "@packages/components/icons";
 import { useI18n } from "@packages/features/i18n-context";
-
 import Section from "./Section";
+import { classes } from "@packages/utils/classes";
 
 export default function WhyItWorksSection() {
   const { t } = useI18n("why-it-works");
   const bgColor = useColorModeValue("gray.50", "gray.900");
```

Comment:
we need to change these colors as well, they are not getting applied

Reply:
We'll need to this for each search that uses chakra colors, or else, we'll deploy a "broken" version to production
- https://www.podcodar.org/ vs https://webapp-git-joel-refsection-component-podcodar.vercel.app/